src/containers/errors/AuthenticationError/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Button } from 'antd';
+import { ReactComponent as Illustration } from '../../assets/images/athentication.svg';
+import './style.css';
+
+const UnAthenticated = () => {
+  return (
+    <div className="error">
```

Comment:
It would be better if you just change the className from error to authentication error because Yosef has a div with the same name.

Reply:
I think it isn't an issue , because error pages are separated and this is a consistent style for naming classes.